appeals/api/src/database/seed/teams/prod.js

```diff
@@ -35,3 +35,354 @@ export const teamsToCreate = [
 	{ id: 17, name: 'North3', email: 'north3@planninginspectorate.gov.uk' },
 	{ id: 18, name: 'North4', email: 'north4@planninginspectorate.gov.uk' }
 ];
+
+export const lpaTeamAssignments = {
```

Comment:
is this duplicated - is the team assignment data not already in seed/LPAs/prod.js ?